transforms/angle-brackets/transform.js

```diff
@@ -263,11 +263,15 @@ function transformNodeAttributes(tagName, node) {
 }
 
 function getDataAttributesFromParams(params) {
-  return params.filter(param => param.original && `${param.original}`.startsWith('data-'));
```

Comment:
I'm wondering if we should explicit here and only allow `PathExpression`

Reply:
I’m cool either way. Want me to change it?

Reply:
if it works, then I'd prefer it :)